articles/policies/rate-limits-mgmt-api.md

```diff
@@ -0,0 +1,97 @@
+---
+# Management API Endpoint Rate Limits
+
+::: warning
+These rate limits apply to you if you subscribed on or after **May 19, 2020**. If you subscribed before **May 19, 2020**, the older [rate limit policy](/policies/legacy-rate-limits) applies to you until **June 18, 2020**. Starting on **June 18, 2020**, these limits will apply to all tenants. You will be notified of the new limits through a **Dashboard Notification**. All existing paid subscriptions that changes will impact will be notified directly via email with additional information about general best practices for minimizing API calls and information about upgrading plans.
+:::
+
+The rate limits for this API differ depending on whether your tenant is free or paid, production or not.
+
+| Tenant Type | Rate Limit (per second) | Rate Limit (per minute) |
+| - | - | - |
+| Free or Trial | 10 | 120 |
+| Enterprise (Non-production) | 10 | 120 |
+| Enterprise (Production) | 50 | 1000 |
+| Developer or Developer Pro (created before 05-21-2020) | 50 | 1000 |
+
+Each endpoint is configured with a bucket that defines:
+
+-  Request limit
+-  Rate limit window (per second, per minute, per day, etc.)
+
+```text
+bucket:
+    size: x
+    per_minute: y
+```
+
+For example, the above states that, for the given bucket, there is a maximum request limit of `x` per minute, and for each minute that elapses, permissions for `y` requests are added back. In other words, for each `60 / y` seconds, one additional request is added to the bucket. This occurs automatically until the bucket contains the maximum permitted number of requests.
+
+For some API endpoints, the rate limits are defined per bucket, so the origins of the call do not influence the rate limit changes. For other buckets, the rate limits are defined using different keys, so the originating IP address is considered when counting the number of received API calls.
+
+::: note
+If you are using an API endpoint **not** listed below and you receive rate limit headers as part of your response, see [Anomaly Detection](/anomaly-detection) for more information.
+:::
+
+The aforementioned rate limits include calls made via [Rules](/rules) and are set **by tenant** and not by endpoint.
+
+The following Auth0 Management API endpoints return rate limit-related headers. For additional information about these endpoints, please consult the [Management API explorer](/api/management/v2).
+
+| Endpoint | Path | Rate Limit (per second) | Rate Limit (per minute) |
+| - | - | - | - |
+| Read or write users | `GET /api/v2/users` | 40 | 800 |
+| | `POST /api/v2/users` | | |
+| Read logs | `GET /api/v2/logs` | 10 | 200 |
+| | `GET /api/v2/user/{id}/logs` | |
+| Read clients | `GET /api/v2/clients` | 5 | 100 |
+| | `POST /api/v2/clients/{id}` | |
+| Read connections | `GET /api/v2/connections` | 10 | 100 |
+| | `POST /api/v2/connections/{id}` | |
+| Delete device credentials | `DELETE /api/v2/device-credentials/{id}` | 5 | 100 |
+| All other endpoints | | 10 | 150 |
```

Comment:
This is only valid for self-service and is already described below, please remove this (also the affected endpoints are not needed).

Reply:
I received some conflicting comments on these tables. I've tried to make the changes as described, but I may need you to review it again.

Reply:
Np, I wil review again.